index.html

```diff
@@ -0,0 +1,341 @@
+			</div>
+			<div class="aside-one-row">
+				<img src="images/a3.png" alt="church">
+				<a
+					href="https://www.nytimes.com/2019/01/11/nyregion/new-utrecht-reformed-church.html?fallback=0&recId=1G4SPPsUZko1MRIdTiNirqBx4mO&locked=0&geoContinent=NA&geoRegion=AB&recAlloc=story-desks&geoCountry=CA&blockId=signature-journalism-vi&imp_id=673155223">Can
+					a Church Founded in 1677 Survive the 21st Century?</a>
+			</div>
+		</div>
+
+		<div class="article-segment-two">
+
+			<p>Confirming inflation would mean that the universe we see, extending 14 billion light-years in space with
+				its hundreds of billions of galaxies, is only an infinitesimal patch in a larger cosmos whose extent,
+				architecture and fate are unknowable. Moreover, beyond our own universe there might be an endless number
+				of other universes bubbling into frothy eternity, like a pot of pasta water boiling over.</p>
+
+			<h3>‘As Big as It Gets’</h3>
+
+			<p>In our own universe, it would serve as a window into the forces operating at energies forever beyond the
+				reach of particle accelerators on Earth and yield new insights into gravity itself. Dr. Kovac’s ripples
+				would be the first direct observation of gravitational waves, which, according to Einstein’s theory of
+				general relativity, should ruffle space-time.</p>
+
+			<p>Marc Kamionkowski of Johns Hopkins University, an early-universe expert who was not part of the team,
+				said, “This is huge, as big as it gets.”</p>
+
+			<p>He continued, “This is a signal from the very earliest universe, sending a telegram encoded in
+				gravitational waves.”</p>
+
+		</div>
+
+		<div id="aside-two">
+			<h5>Sign up for Science Times</h5>
+			<p>We’ll bring you stories that capture the wonders of the human body, nature and the cosmos.</p>
+			<input type="button" value="SIGN UP">
+		</div>
+
+		<div class="article-segment-three">
+
+			<p>The ripples manifested themselves as faint spiral patterns in a bath of microwave radiation that
+				permeates space and preserves a picture of the universe when it was 380,000 years old and as hot as the
+				surface of the sun.</p>
+
+			<p>Dr. Kovac and his collaborators, working in an experiment known as Bicep, for Background Imaging of
+				Cosmic Extragalactic Polarization, reported their results in a scientific briefing at the Center for
+				Astrophysics here on Monday and in a set of papers submitted to The Astrophysical Journal.</p>
+
+			<h4>The Theory of Inflation</h4>
+
+			<p class="main-pic-caption">Astronomers have found evidence to support the theory of inflation, which
+				explains how the universe expanded so uniformly and so quickly in the instant after the Big Bang 13.8
+				billion years ago.</p>
+
+			<img alt="nyt-article" src="https://github.com/GT001/TOP-NYT-Article/blob/master/images/TOI.png?raw=true" width="100">
+			<p id="main-pic-caption-name"><span id="line-above">By LARRY</span> BUCHANAN and JONATHAN CORUM</p>
```

Comment:
Make sure the width of this image matches the original design

Reply:
Hi, @lily-coder ✋  I think it is important to make an effort and try to compose your message in a way that has a friendly overtone. I think we don’t want to be misunderstood or ruin somebody’s day with an imprecise message. If you want people to perceive your messages right, please make suggestions instead of giving orders. For example, you can say here:

**The width of the image is not matching the original design, Please check the original article given in the project requirement. You can check this [link ](https://phancybox.co.nz/news/find-dimensions-of-image-on-website/)for more information 🏃 .**